kafka-ui-react-app/src/components/Connect/List/__tests__/List.spec.tsx

```diff
@@ -47,6 +47,7 @@ describe('Connectors List', () => {
                 areConnectsFetching
                 connectors={[]}
                 connects={[]}
+                failed={[]}
                 fetchConnects={fetchConnects}
```

Comment:
we can actually pass failed parameter to the , setUpComponent and test the failed rendering view. so we can make it here conditional.